### DISCUSSION TITLE
Garage Level Supplies vs Car Level Supplies

### DISCUSSION THREAD

**Discussion by varunpan** [hargata/lubelog#409] [Q&A] [answered]
What is the difference between Garage Level Supplies and the Car Level Supplies. I cannot see the Garage Level Supplies in the car level supplies and vice versa? What are they for?

**Comment by hargata** [comment, marked as answer]
You can see the garage level supplies for all vehicles when you go to "Choose Supplies" for the record. Vehicle level supplies are only available for that specific vehicle.

![image](https://github.com/hargata/lubelog/assets/155338622/05b8fdd8-21e1-4be2-84b1-6cd42678f9f4)

**Reply by varunpan** [reply]
Ok understood. Should I be choosing a supply that is supposed to be a tool and not consumable when creating a service record?

**Reply by hargata** [reply]
You can if you want to, but it's not designed to be used that way. The supplies functionality was designed for consumables, and every time it's used in a record, it will adjust the remaining quantities accordingly.

I know there are users that just adds a tool with an impossibly high amount of quantity such as 999999 and they just use it similarly to a consumable supply.

**Reply by varunpan** [reply]
Ahh ok that makes sense. I wanted to track what tools and supplies were used in the services for record keeping
